administrator/components/com_content/Model/ArticlesModel.php

```diff
@@ -205,9 +207,9 @@ protected function getListQuery()
 		$query->select(
 			$this->getState(
 				'list.select',
-				'a.id, a.asset_id, a.title, a.alias, a.checked_out, a.checked_out_time, a.catid' .
-				', a.state, a.access, a.created, a.created_by, a.created_by_alias, a.modified, a.ordering, a.featured, a.language, a.hits' .
-				', a.publish_up, a.publish_down, a.introtext, a.note'
+				'DISTINCT a.id, a.asset_id, a.title, a.alias, a.checked_out, a.checked_out_time, a.catid' .
```

Comment:
```suggestion
				'a.id, a.asset_id, a.title, a.alias, a.checked_out, a.checked_out_time, a.catid' .
```
The distinct has recently been removed in 4.0-dev branch, together with changing the subquery. @SharkyKZ is that correct?